plugins/woocommerce/client/blocks/packages/components/panel/style.scss

```diff
@@ -1,16 +1,14 @@
 .wc-block-components-panel.has-border {
-	@include with-translucent-border(1px 0);
+	border: 1px solid $universal-translucent-border-color;
+	border-width: 1px 0;
 
 	+ .wc-block-components-panel.has-border::after {
 		border-top-width: 0;
 	}
```

Comment:
Looks like this was trying to remove the top border, but now it won't because the `after` has been removed, looks like line 3 could be updated to set a top width explicitly to 0 can't it?